primefaces/src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -22851,11 +22851,11 @@
         </attribute>
         <attribute>
             <description>
-                <![CDATA[]]>
+                <![CDATA[Defines the maximum height of the scrollable area.]]>
             </description>
             <name>scrollHeight</name>
             <required>false</required>
-            <type>java.lang.Integer</type>
+            <type>java.lang.String</type>
```

Comment:
We should mention this in the migration guide, or accept Object. @melloware WDYT?

Reply:
@melloware so not change to Object?

Reply:
I think he made it consistent with how SelectOne Menu is so we should make it consistent. And since its a change in property type we should mention it in 13.0/migration guide.